code/game/gamemodes/changeling/powers/vitamin_synthesis.dm

```diff
@@ -0,0 +1,18 @@
+//Vitamin Synthesis: Allows the changeling to shift their vitamin levels at will.
+/obj/effect/proc_holder/changeling/vitamins
+	name = "Vitamin Synthesis"
+	desc = "Allows us to shift our blood nutrient levels at will. Higher vitamin concentrations supplement the body's natural healing ability and allow faster recuperation of conventional wounds."
```

Comment:
changeling power that give it vitamins, where it talks about the various health benefits of eating healthy

i'm dead